lib/linter-eslint.coffee

```diff
@@ -31,6 +31,15 @@ module.exports =
       type: 'string'
       default: ''
       description: 'Run `$ npm config get prefix` to find it'
+    fileScopes:
+      type: 'string'
```

Comment:
This should be an [array](https://atom.io/docs/api/v1.1.0/Config#array) or strings.